### DISCUSSION TITLE
中央需要有的「連鎖飲料店」

### DISCUSSION THREAD

**Discussion by LinChloe** [1101-GS4719-C/Python-Community-GS4719-C#1951] [General]
來了中央後真的有嚇到，這裡的連鎖飲料店少的可怕，原本以為有大學生這種愛喝飲料的族群，飲料店應該不會少。
沒想到事與願違，最後只能和朋友請外送，等很久又要多付錢。(想哭QAQ)

本人希望能在中央有的飲料店(心中連鎖飲料店排名):
**1. 清源**
身為芋頭控，芋泥和芋圓我都不放過。
認真覺得清源的芋圓好吃，很Q、芋頭為夠、也不會有放很久的感覺。
![清](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/5dbe7f1d-73e2-4a92-9809-735172a6e662)

**2.得正**
茶類中我比較喜歡喝烏龍茶，之前喝了他們家的重焙烏龍後，直接搶走了我的心。
而且價格也不貴，下課後買杯犒賞自己，在適合不過了!
![得](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/e1708403-8914-429a-9c6f-dd89842907b8)

**3.麻古**
奶蓋真的是一個偉大的發明，奶蓋搭配清爽的茶類飲料，在濃郁與清爽間碰撞出無法抵擋的魅力。
麻古的芝芝對我來說還不錯~
(話說其實我比較喜歡鹹奶蓋)
![麻](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/68bb1802-eb58-46f0-a787-cfe6e5a9a4df)

請各位說出你心中最偉大的飲料店，並向大眾推廣它吧!!
(可以的話推薦下鹹奶蓋的飲料店，等暑假我會去踩點的~)

**Comment by 710388** [comment]
得正飲料店

**Comment by willychiuchiu** [comment]
50嵐！國高中時期最愛喝他們的1號（四季春珍波椰），絕對是經典中的經典，下課都一定要來一杯。但上了中央以後五十嵐都要跑大老遠才買得到，就變得超級少喝了，不過這樣也好戒飲料也順便省錢

**Comment by 110101501** [comment]
同樣也是得正愛好者！喜歡喝純茶類的飲料～春烏龍深得我心，喝起來很清甜
除此之外，也很喜歡五十嵐的烏龍奶茶配小珍珠，每次給的料都很多，喝起來很滿足！

**Comment by jeff110303549** [comment]
確實能夠對版主想法感同身受，中央附近的飲料店實在屈指可數的少，既然已經無法改變學校蓋在一個偏僻的地方，至少往好處想，我們學校不遠的周圍也是有許多值得一提的地方

**Comment by woohsin** [comment]
我在家的時候幾乎一天一杯五十嵐，強烈希望出現五十嵐！
還有麻古、茶湯會都不錯，奶茶都好喝！

**Comment by hbh10601** [comment]
茶湯會跟澄夏
茶湯會是春水堂小品牌
而澄夏是主打新鮮果汁類的 個人很推
不過比起飲料店 更希望有一個正式經營的好喝的酒吧

**Comment by YuanWang08** [comment]
中央附近就跟廢墟一樣，校內有王者香、宵夜街還有三間，後門有清心，我覺得已經非常多了。他們大多都是做學生生意，如果飲料店再多我覺得會有幾間倒掉，畢竟這破地方就這麼小。

**Comment by 20050201Shawn** [comment]
我沒有特別愛喝啥但通常都會點珍珠鮮奶茶，比較常喝拾玖茶屋的珍珠鮮奶(便宜味道又還行)，by the way圖片裡的飲料看起來好好喝。

**Comment by sunset513** [comment]
我也很喜歡版主推的第二和第三名，這兩間的口味很對我的胃口。

我自己私心想要的店是「一手私藏世界紅茶」，這間的茶香味很剛好，搭配上不會太甜的珍珠，適合我這種口味清淡的人。

**Comment by wanka2030** [comment]
之前在讀高中的時候清原就有在我們學校營業，他的營收好像也不錯(至少下午的是後會是賣光的，但中央里面的店家真的選擇太少了，而且多數為自主開店，如果能夠增加一些連鎖店應該會不錯:D

**Comment by happy246810** [comment]
我覺得中央附近的飲料連鎖店是真的很少，可能是因為中央的地理位置真的比較偏遠，導致沒有飲料店願意進駐，如果要我推的話，應該會是紅茶洋行，因為他的紅茶真的不錯喝，但可惜的是他只有開在東部和北部。

**Comment by LinChloe** [comment]
為什麼都沒人推薦鹹奶蓋給我，難道只有我喜歡鹹奶蓋嗎?
鹹鹹的奶蓋很好吃的說。

**Comment by YifenLin** [comment]
我很少喝飲料，在朋友的推薦下喝過一次得正，我覺得很好喝ㄟ，濃濃奶蓋的配上清爽的烏龍茶超爽口消暑，適合一邊逛街一邊喝。不過怕胖的我還是希望學校附近不要開太多飲料店，我怕我一個不小心就克制不住自己買飲料的慾望哈哈。

**Comment by SusuTima** [comment]
清原一票，我想吃「芋見幸福」。

**Comment by thomasxshen** [comment]
可不可超好喝，以前我每天下課都會去喝一杯他的熟成紅茶，也成為了我高中時代的寶貴回憶，希望中央能開。

**Comment by yyujimmy** [comment]
這幾家飲料店看起來都很不錯！我也是芋頭控，清源的芋圓和芋泥聽起來真的很吸引人。而得正的重焙烏龍茶和麻古的奶蓋茶也讓人垂涎欲滴呢！

我最喜歡的飲料店是「鮮茶道」，他們的鮮奶茶非常好喝，茶味濃郁、奶香十足，加上適中的甜度，每一口都是享受！另外，如果你喜歡鹹奶蓋，可以嘗試他們的鮮奶蓋茶系列，絕對不會讓你失望的。

希望你有機會在暑假時去踩點，品嚐一下鮮茶道的美味！也期待你在中央找到更多喜歡的飲料店，不再需要等外送了！

**Comment by poiu6fi** [comment]
想喝在台中的八曜合茶

**Comment by Tchang323** [comment]
確實，許願發發牧場
她們的優格配紫米真的好好喝!!
